app/screens/global_threads/threads_list/thread/thread.tsx

```diff
@@ -0,0 +1,239 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {Text, TouchableHighlight, View} from 'react-native';
+
+import {fetchAndSwitchToThread} from '@actions/remote/thread';
+import FormattedText from '@components/formatted_text';
+import FriendlyDate from '@components/friendly_date';
+import {Screens} from '@constants';
+import {useServerUrl} from '@context/server';
+import {useIsTablet} from '@hooks/device';
+import {bottomSheetModalOptions, showModal, showModalOverCurrentContext} from '@screens/navigation';
+import {preventDoubleTap} from '@utils/tap';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+import {typography} from '@utils/typography';
+import {displayUsername} from '@utils/user';
+
+import ThreadFooter from './thread_footer';
+
+import type ChannelModel from '@typings/database/models/servers/channel';
+import type PostModel from '@typings/database/models/servers/post';
+import type ThreadModel from '@typings/database/models/servers/thread';
+import type UserModel from '@typings/database/models/servers/user';
+
+type Props = {
+    author?: UserModel;
+    channel?: ChannelModel;
+    post: PostModel;
+    teammateNameDisplay: string;
+    testID: string;
```

Comment:
testID does not really need to be a prop.  It can be constructed in the component itself.  Is that right?

Reply:
`global_threads.list` is passed as a prop and used as a prefix from `global_threads.tsx`